pull_data.R

```diff
@@ -0,0 +1,33 @@
+library(sbtools)
+library(data.table)
+library(readxl)
+
+source("auth.R")
```

Comment:
`R_HOME` has been used before (hazardItems, necsc-lake-modeling, etc), which wouldn't be as at risk of someone adding accidentally to public source control.